_posts/2023-11-20-distributed-realm-ignore-unavailable-realms.adoc

```diff
@@ -0,0 +1,9 @@
+---
+layout: post
+title: 'Using distributed realm ignore-unavailable-realms attribute in Elytron'
+date: 2023-11-20
```

Comment:
@lvydra Thanks for the PR! Would you be able to update the date in the filename and here?